### ISSUE TITLE
Python 3 only

### ISSUE THREAD

**Issue by jayvdb** [coala/coala-vs-code#35]
If it wasnt already, https://github.com/coala/coala-vs-code/pull/34 made this project Python 3.4+ only.

That should be documented in the README.md.

**Comment by jkaytser** [comment]
can this be assigned to me?

**Comment by li-boxuan** [comment]
That's definitely true, I should have mentioned that. IMO coala requires Python >= 3.4, so this won't be a problem. But for sure, it should be documented.

**Comment by gaocegege** [comment]
Yeah, please follow [the guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to be the member of coala, then we could assign the issue to you.

**Comment by sagban** [comment]
Yes I followed the guide. Now i have to fixed the newcomer issue.

**Comment by gaocegege** [comment]
You need to accept the invitation to be the member of coala org in GitHub to allow us to assign the issue to you.

**Comment by sagban** [comment]
I followed the gitter chat room link. Now what else i have to do?

**Comment by gaocegege** [comment]
http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-1-meet-the-community-and-get-an-invitation-to-the-organization

I think you should reveive a email about the invitation.

**Comment by sagban** [comment]
No I didn't get the email. I checked it.
Should I type Hello Word in the chat?

**Comment by gaocegege** [comment]
Then @jayvdb Could you invite @sagban ?
